lib/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Authorizer } from './authorizer';
+import { ExpressRequest } from './interfaces';
+
+export default function(password: string) {
+  const authorizer = new Authorizer(password);
+
+  const middleware = function(request: ExpressRequest, response: any, next) {
+    authorizer.getFromAuthorizationHeader(request);
+    const auth = request.logentriesWebhookAuth;
+
+    if (!auth || !auth.user) {
```

Comment:
It is customary in expressjs for an object representing the authenticated user to be put into the user property of the request object. This for instance is where passportjs puts an object representing the user, and is typical in sample client code using information about an user authenticated by expressjs middleware.

While I don't care much for mutating the state of the request object in the getter function, I think it's wholly appropriate and typical to expect middleware functions to mutate the state of the request object.

Reply:
I'd agree that this would have been the better (more standardized) approach. On this issue though I'd prefer to leave the external interface of the code as is to not introduce another breaking change. Unless you think it's worth it?

Reply:
Nah, leaving it alone is fine for now

Reply:
👍